docs/user-guide/src/capm3/introduction.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Cluster API Provider Metal3
+
+Kubernetes-native declarative infrastructure for Metal3.
+
+<div>
+    <img src="../images/metal3-color.svg" width="120px" />
+</div>
+
+## What is the Cluster API Provider Metal3
+
+The [Cluster API](https://github.com/kubernetes-sigs/cluster-api/) brings declarative,
+Kubernetes-style APIs to cluster creation, configuration and management. The API
+itself is shared across multiple cloud providers. Cluster API Provider Metal3 is
+one of the providers for Cluster API and enables users to deploy a Cluster API based
+cluster on top of bare metal infrastructure using Metal3.
+
+## Compatibility with Cluster API
+
+| CAPM3 version | Cluster API version | CAPM3 Release |
+|---------------|---------------------|---------------|
+| ~~v1alpha4~~  | ~~v1alpha3~~        | ~~v0.4.X~~    |
+| v1alpha5      | v1alpha4            | v0.5.X        |
+| v1beta1       | v1beta1             | v1.1.X        |
```

Comment:
Maybe we do not need this info here at all since we do have a https://deploy-preview-280--metal3io.netlify.app/version_support.html page where we outline the supported versions in detail, WDYT?

Reply:
Actually no Furkat, they are talking about different things, supported versions talks about what are the current versions we are supporting. I am trying to highlight here the compatibility of CAPM3 with CAPI. I agree that supported version include a link to compatibility but that includes IPAM as well. However since this is CAPM3 intro, I would like to keep this section here.

Reply:
That makes sense, let's keep it as is then. We do have CAPM3 v1.2.X releases though since this has been last updated, can we include it in the list, please?